package.json

```diff
@@ -37,7 +37,10 @@
     "husky": "^3.1.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-scripts": "^3.2.0",
+    "react-use": "^13.10.0",
     "semantic-release": "^15.13.31",
+    "styled-components": "^4.4.1",
```

Comment:
do we use those?

Reply:
We use them in the docs: https://github.com/ModaOperandi/icons/blob/fa52f5e3d8f331b5692bc182a159d480801efb94/example/package.json#L11-L13

So they shouldn't be in this package.json.

Was this added in an effort to get the build passing?

Reply:
yeah it's used for the example page, see `example/src/App.js`

Reply:
> We use them in the docs:
> 
> https://github.com/ModaOperandi/icons/blob/fa52f5e3d8f331b5692bc182a159d480801efb94/example/package.json#L11-L13
> 
> So they shouldn't be in this package.json.
> 
> Was this added in an effort to get the build passing?

Oh ok that makes sense.
I couldn't get `yarn local:preview` working locally that's why I added them. I'll remove these.

Reply:
Ah yeah; my bad; I need to add an install step in there.